### PR TITLE
Adds a contributors page

### PR DESCRIPTION
**Pull request by dm36**
Adds a contributors page and adds instructions on how to run design-system-web to the README.

### CODE REVIEW

File: UXWContributors.js [hunk 1]
```diff
@@ -1,13 +1,77 @@
...
-        Greedo argazdan jinn tof verpine bothan wedge.
-      </p>
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <style dangerouslySetInnerHTML={{__html: "\nhtml {\n  box-sizing: border-box;\n}\n\n*, *:before, *:after {\n  box-sizing: inherit;\n}\n\n.column {\n  float: left;\n  width: 33.3%;\n  margin-bottom: 16px;\n  padding: 0 8px;\n}\n\n@media screen and (max-width: 650px) {\n  .column {\n    width: 100%;\n    display: block;\n  }\n}\n\n.card {\n  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2); font-size:12px\n}\n\n.container {\n  padding: 0 16px;\n}\n\n.container::after, .row::after {\n  content: \"\";\n  clear: both;\n  display: table;\n}\n\n.title {\n  color: grey;\n}\n\n.button {\n  border: none;\n  outline: 0;\n  display: inline-block;\n  padding: 8px;\n\n  color: white;\n  background-color: #6331FD;\n  text-align: center;\n  cursor: pointer;\n  width: 100%;\n}\n\n.button:hover {\n  background-color: #555;\n}\n" }} />
+      <h1>Meet The Ux Wizards!</h1>
+      <List>
+      {data.map(({name, role, email, img}) => {
```
Comment (kaisenl) [inline UXWContributors.js:59]: `{data.map(({name, role, email, image}) => (`

File: UXWContributors.js [hunk 2]
```diff
@@ -1,13 +1,77 @@
...
+        <div className="row">
+        <div className="column">
+          <div className="card">
+            <img src={img} alt="Jane" style={{width: '100%'}} />
```
Comment (kaisenl) [inline UXWContributors.js:63]: `image`

File: UXWContributors.js [hunk 3]
```diff
@@ -1,13 +1,77 @@
...
+            <div className="container">
+              <h2>{name}</h2>
+              <p className="title">{role}</p>
+              <p>Some text that describes me lorem ipsum ipsum lorem.</p>
+              <p>{email}</p>
+              <p><button className="button">Contact</button></p>
+            </div>
+          </div>
+        </div>
+        </div>})}
```
Comment (kaisenl) [inline UXWContributors.js:73]: `</div>))}`

File: README.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
 # UX Wizards Design System
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/56b1c0d0-a521-4c51-8afd-0faf409a5305/deploy-status)](https://app.netlify.com/sites/uxwizards-designsystem/deploys)
+
+# Design System Web
+
+To run design-system-web locally:
+
+```
+php -S localhost:9000
```
Comment (javag97) [inline README.md:10]: **Praise**: Docs!

File: UXWContributors.js [hunk 5]
```diff
@@ -1,13 +1,104 @@
...
     <div className="container">
-      <p>
-        Lucas ipsum dolor sit amet carnor needa chazrach rebo sebulba plagueis finis di veknoid ventress.
-        Maul lando vader sy hutt nomi. Gordin haruun bail jar. Fett mothma toydarian jerec greedo bothawui sesswenna artaru zuckuss.
-        Nassau cerea sern naboo. Tagge leia obi-wan thennqora antilles. Davin selkath jobal tavion metalorn organa muunilinst jettster.
-        jinn hutt ikrit vor. Bertroff kael hutt wharl oola dagobah bibble saleucami. Colton raioballo ansionian hapan billaba wampa organa.
-        Greedo argazdan jinn tof verpine bothan wedge.
-      </p>
+      <meta name="viewport" content="width=device-width, initial-scale=1" />
+      <h1>Meet The Ux Wizards!</h1>
```
Comment (javag97) [inline UXWContributors.js:84]: **Small Suggestion**: Change Ux → UX

File: UXWContributors.js [hunk 6]
```diff
@@ -1,13 +1,104 @@
...
+      <List>
+      {data.map(({name, role, email, image}) => (
+      <div className="row">
+        <div className="column" style={columnStyle}>
+          <div className="card" style={cardStyle}>
```
Comment (javag97) [inline UXWContributors.js:89]: **Suggestion**(non-blocking): I would consider making the whole card clickable if you're looking to add a contact button to it.

File: UXWContributors.js [hunk 7]
```diff
@@ -1,13 +1,104 @@
...
+  },
+  {
+    name: 'Dhruv',
+    role: 'Software',
+    email: 'example@example.com',
+    image: '/assets/dhruv.jpg'
+  },
+]
+
+const buttonStyle = {
```
Comment (javag97) [inline UXWContributors.js:52]: I Will always encourage you to use tailwindCSS 😄 , but this is ok to me.

File: UXWContributors.js [hunk 8]
```diff
@@ -1,13 +1,104 @@
...
+const columnStyle = {
+  float: "left",
+  width: "33.3%",
+  "margin-bottom": "16px",
+  padding: "0 8px"
+}
+
+const cardStyle = {
+  "box-shadow": "0 4px 8px 0 rgba(0, 0, 0, 0.2)",
+  "font-size": "10px"
```
Comment (javag97) [inline UXWContributors.js:77]: **Suggestion**(non-blocking): Text is rather small IMO. I would default to [body text being one root em](https://www.w3.org/Style/Examples/007/units.en.html). Browsers default this value to be `16px`, so I would start here and make adjustments from this baseline

**Review by javag97 — CHANGES_REQUESTED** [review]

**Review by javag97 — APPROVED** [review]
Approval! We can collaborate on TailwindCSS in another PR :-)

**Pull request merged**